sourced/ml/algorithms/uast_to_role_id_pairs.py

```diff
@@ -0,0 +1,75 @@
+from typing import Iterable, Tuple
+
+import bblfsh
+
+from sourced.ml.algorithms.uast_ids_to_bag import UastIds2Bag
+
+
+class MergeRoles:
+    def __getitem__(self, roles):
```

Comment:
I would rather made this a simple function and change `roles2index` type from `[]` to `callable`. Saves time to understand.

Reply:
yes, I agree, but we should change it here too: https://github.com/src-d/ml/blob/master/sourced/ml/algorithms/uast_ids_to_bag.py#L28 (an FakeVocabulary: https://github.com/src-d/ml/blob/master/sourced/ml/algorithms/uast_ids_to_bag.py#L9 )

I code like this for uniformity.

Should I change it in both places or keep as is?

Reply:
Change it only here and insert FIXME for the rest, do not diverge to refactoring now.